src/Microcks.Testcontainers/MicrocksContainerEnsemble.cs

```diff
@@ -221,6 +228,24 @@ public MicrocksContainerEnsemble WithAmqpConnection(GenericConnection amqpConnec
         return this;
     }
 
+    /// <summary>
+    /// Enables DEBUG log level for the containers of this ensemble.
+    /// </summary>
+    /// <remarks>
+    /// Applies to the main Microcks container (Spring) and, when the async feature is enabled,
+    /// to the async-minion container (Quarkus).
+    /// </remarks>
+    public MicrocksContainerEnsemble WithDebugLogLevel()
+    {
+        this._debugLogLevelEnabled = true;
+        this._microcksBuilder.WithDebugLogLevel();
```

Comment:
In TestContainers, Builders are immutable: WithDebugLogLevel() returns a new instance, so we must reassign it.
```suggestion
        this._microcksBuilder = this._microcksBuilder.WithDebugLogLevel();
```